rogue-thi-app/lib/backend-utils/rooms-utils.js

```diff
@@ -205,6 +205,22 @@ export async function findSuggestedRooms (room, startDate, endDate) {
   return rooms
 }
 
+export async function getEmptySuggestions () {
```

Comment:
It would be greatly appreciated if you could enhance the code's readability and maintainability by adding comprehensive documentation to your function. This will provide a clear understanding of its purpose, input parameters, and expected output, making it easier for others to collaborate and maintain the code in the future.  :))